### ISSUE TITLE
how to specify a location with demo.py ?

### ISSUE THREAD

**Issue by blueskybreeze** [rubenvereecken/pokemongo-api#89]
is the following line supposed to work?

`python demo.py -a "google" -u "email@gmail.com" -p "thepassword" -l "34.567,-123.456"`

It won't show any nearby pokemons... but in my Pokemon Go app I do see some nearby pokemons.

**Comment by dnsBlah** [comment]
use encrypt.so

**Comment by blueskybreeze** [comment]
where should encrypt.so be placed on a Mac?  I placed it on all possible folders in `pokemongo-api` but it didn't work.

  I only saw the instruction for Pokemon Go Bot:  

http://blog.pokemongofun.com/tag/pokemon-go-encrypt-so/

but I am not running a bot... (do I need to run it

**Comment by dnsBlah** [comment]
just place it in the same folder of demo.py
and call the script with -e "filename.extension"

**Comment by blueskybreeze** [comment]
do you mean

`python demo.py -a "google" -u "email@gmail.com" -p "thepassword" -l "37.7954961,-122.3935113" -e "encrypt.so"`

?

**Comment by blueskybreeze** [comment]
also by the way, I am using Python 2.7.5 and I saw many lines that said I might need a newer version of Python -- not sure if it matters:

/Users/mike/.virtualenvs/django19try01/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#snimissingwarning.
  SNIMissingWarning

**Comment by dmadisetti** [comment]
did you install from requirements?

**Comment by blueskybreeze** [comment]
after `git clone` (recursive or not)... I did run a

`pip install -r requirements.txt`

**Comment by dnsBlah** [comment]
please uninstall urllib and reinstall it

**Comment by blueskybreeze** [comment]
is it by `pip uninstall urllib` ?  But it will say

```$  pip uninstall urllib
Cannot uninstall requirement urllib, not installed```

**Comment by blueskybreeze** [comment]
I also tried

`pip uninstall -r requirements.txt`

and then

`pip uninstall -r requirements.txt`

but it didn't work.

**Comment by dmadisetti** [comment]
[Create new virtual env](https://virtualenv.pypa.io/en/stable/), then `pip install .` then try?

**Comment by ealejand** [comment]
To get rid of the SNIMissingWarnings you can `pip install requests[security]`

**Comment by dnsBlah** [comment]
cool! thanks ealehand
is this something with the urllib version or something?

**Comment by ealejand** [comment]
Yeah, from what I've read Python 2 lacks SNI support so there's a couple packages that can supplement certificate validation.

http://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
